apps/frontend/src/users/Profile.tsx

```diff
@@ -28,7 +38,19 @@ const Page: React.FC<{}> = () => {
           team={'Team Unknown'}
           title={'Unknown Title'}
           role={'Unknown Role'}
+          aboutHref={join(url, 'about')}
+          researchInterestsHref={join(url, './research-interests')}
+          outputsHref={join(url, 'outputs')}
         />
+        <Switch>
+          <Route path={`${path}/about`} render={() => 'About'} />
+          <Route
+            path={`${path}/research-interests`}
+            render={() => 'Research interests'}
+          />
+          <Route path={`${path}/outputs`} render={() => 'Outputs'} />
+          <Redirect to={join(url, 'about')} />
```

Comment:
I was imagining we would keep about as the default route (render it on `/`). This is also an option but do we need the extra redirect?

Reply:
I don't have a strong opinion on this, only reason I did it this way was symmetry / consistency

Reply:
Relative links can be a bit confusing when tabs are at different levels, but with the `join` structure I used it should be fine

Reply:
I guess a third option would be having the default tab as a "fallback" non-exact third route with path `/`, so that both `/` and `/about` (and `/whatever`) match it.
Not something I prefer, just naming it for completeness

Reply:
Let me know whether you want this changed or you're equally unpassionate about it :sweat_smile: